components/Article/EditLink/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPencil } from '@fortawesome/free-solid-svg-icons';
+import styles from './index.module.scss';
+import { useLocale } from './../../../hooks/useLocale';
+
+interface Props {
+  relativePath?: string;
+  editPath?: string;
+  absolutePath?: string;
+}
+
+// TODO(HinataKah0): Change this
+const baseURL =
+  'https://github.com/nodejs/nodejs.org/edit/major/website-redesign';
```

Comment:
What will happen if we don't have translated files here? If we use this URL, this will directly guide us to change the English ones, and this won't be a right one.

If we don't have a translated file, the URL should be something like this:

https://github.com/nodejs/nodejs.org/new/major/website-redesign （Change "edit" to "new" in this URL）.